pkgs/by-name/au/autokuma/package.nix

```
@@ -0,0 +1,39 @@
+{
+  lib,
+  fetchFromGitHub,
+  rustPlatform,
+  pkg-config,
+  openssl,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "AutoKuma";
```

```suggestion
  pname = "autokuma";
```

better to align with attribute name so it's easier to find in the store